README.md

```diff
@@ -59,6 +59,11 @@ As noted in our [security policy](https://github.com/newrelic/.github/blob/maste
 
 If you believe you have found a security vulnerability in this project or any of New Relic's products or websites, we welcome and greatly appreciate you reporting it to New Relic through [HackerOne](https://hackerone.com/newrelic).
 
+If you would like to contribute to this project, please review [these guidelines](https://github.com/newrelic/newrelic-dotnet-agent/blob/main/CONTRIBUTING.md).
```

Comment:
The URL can be simplified to just `(/CONTRIBUTING.md)`.

Reply:
Updated.